load_balancing.go

```diff
@@ -66,6 +66,7 @@ type LoadBalancer struct {
 	RegionPools    map[string][]string `json:"region_pools"`
 	PopPools       map[string][]string `json:"pop_pools"`
 	Proxied        bool                `json:"proxied"`
+	Enabled        *bool               `json:"enabled,omitempty"`
```

Comment:
What was the thinking here as to using a pointer? We always have a value provided in the API responses I've seen so this should be a `bool` (not `*bool`).

If it's in regards to making the tests green, you just needed to add the `enabled` values to the test and it should be golden.

Reply:
No, it about not omitting false values, which I was worried about when I created the original PR (and why I did not use omitempty and made it fail the tests).

There's precedence in zone.go:

commit ea9272e4235ff7a9aa37e2c7c7a8debe22b3d696
Author: James O'Gorman <james@netinertia.co.uk>
Date:   Sun Feb 19 20:42:38 2017 +0000

    fix(zone): fix the ability to un-pause zones

    Due to using `omitempty` to not send unset values to the API, this also
    meant that `false` values for `paused` would also not be sent.

    Using a pointer instead fixes this issue.

    Closes #39

Reply:
I'm not entirely sure if that's the same issue here as the "enabled" value is always present and uses a PUT endpoint for updates (read full replacement) whereas the zone update endpoint is PATCH (partial updates).

Reply:
When using a simple bool in the struct, if I don't use omitempty, then I introduce "enabled: false" to every LoadBalancer struct we don't explicitly set the enabled field.  This is why the original tests failed and is not desirable.  If anything we'd like it to default to true.  If I do use omitempty with a simple bool, then false values won't be passed onto the API as they will be omitted, so we can never disable a load balancer.

I think it has to be a pointer to allow for the values of unset, true and false. 

I'm not a native go programmer so someone more familiar may wish to revue this.

Reply:
Gotcha, it's the default values causing pain here. There are a couple of ways to Do This Better but I think we'd look to apply that across the board in one fell swoop instead of dribs and drabs.